soh/include/z64save.h

```diff
@@ -25,6 +25,12 @@ typedef struct {
     /* 0x5C */ s16 gsTokens;
 } Inventory; // size = 0x5E
 
+typedef struct {
+    /*      */ u8 heartPieces;
+    /*      */ u8 heartContainers;
+    /*      */ u8 dungeonKeys[19];
+} Stats;
```

Comment:
Stats because I imagine this is where we will store other stat related things like bonks, time played, damage taken, etc.

Reply:
This struct may be a better place to store visited entrances/scenes as well 👀 for the entrance tracker I'm working on.